docs/src/components/ComponentDoc/ComponentDocAccessibility.tsx

```diff
@@ -21,17 +29,89 @@ function getStem(info) {
 }
 
 function getBehaviorName(stem) {
-  const filename = stem && `${stem}.ts`
+  if (!stem) {
+    return undefined
+  }
+  const filename = stem && `${stem}.ts`.replace(stem[0], stem.charAt(0).toLowerCase())
+  for (let i = 0; i < behaviorMenu.length; i++) {
+    const behaviour = behaviorMenu[i].variations.find(
+      behaviorVariation => behaviorVariation.name === filename,
+    )
+    if (behaviour) {
+      return behaviorMenu[i].displayName
+    }
+  }
+}
+
+function getId(idName: string) {
+  return _.kebabCase(idName)
+}
+
+function getAvailableBehaviorsFromJson(availableBehaviors: any): any[] {
+  const availableBehaviorsFromJson = []
+  availableBehaviors.map(availableBehavior => {
+    behaviorMenu.map(behaviorInJson => {
+      const result = behaviorInJson.variations.find(
+        variation => variation.name === `${availableBehavior.name}.ts`,
+      )
+      if (result) {
+        availableBehaviorsFromJson.push(result)
+      }
+      return
+    })
+    return
+  })
+  return availableBehaviorsFromJson
+}
+
+function getAccIssues(info) {
+  return _.get(_.find(info.docblock.tags, { title: 'accessibilityIssues' }), 'description')
+}
 
-  return behaviorMenu.reduce((acc, next) => {
-    return _.find(next.variations, { name: filename }) ? next.displayName : acc
-  }, null)
+const baseName = (fileName: string) => {
+  const divided = _.startCase(fileName.replace(/Behavior\.ts$/, ''))
+  return _.upperFirst(_.lowerCase(divided))
+}
+
+const behaviorCard = (variation: any, keyValue: string) => {
+  return (
+    <React.Fragment key={keyValue}>
+      <Segment className="docs-example" id={getId(variation.name)} styles={exampleStyle}>
+        <ComponentExampleTitle
+          title={baseName(variation.name)}
```

Comment:
Same thing, is used once, can be just variable inside component

Reply:
baseName function I moved now to "BehaviorCard.tsx" file. 
I let the logic there, as it looks for me a bit tricky to put it inline.
@layershifter  what do you think about? :)